FND/AllFND-NorthAmerica.rdf

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE rdf:RDF [
+	<!ENTITY dct "http://purl.org/dc/terms/">
+	<!ENTITY fibo-fnd-all "https://spec.edmcouncil.org/fibo/ontology/FND/AllFND/">
+	<!ENTITY fibo-fnd-plc-uspsa "https://spec.edmcouncil.org/fibo/ontology/FND/Places/NorthAmerica/USPostalServiceAddresses/">
+	<!ENTITY fibo-fnd-plc-uspsai "https://spec.edmcouncil.org/fibo/ontology/FND/Places/NorthAmerica/USPostalServiceAddressesIndividuals/">
+	<!ENTITY fibo-fnd-utl-av "https://spec.edmcouncil.org/fibo/ontology/FND/Utilities/AnnotationVocabulary/">
+	<!ENTITY fibo-fndna-all "https://spec.edmcouncil.org/fibo/ontology/FND/AllFND-NorthAmerica/">
+	<!ENTITY owl "http://www.w3.org/2002/07/owl#">
+	<!ENTITY rdf "http://www.w3.org/1999/02/22-rdf-syntax-ns#">
+	<!ENTITY rdfs "http://www.w3.org/2000/01/rdf-schema#">
+	<!ENTITY skos "http://www.w3.org/2004/02/skos/core#">
+	<!ENTITY sm "http://www.omg.org/techprocess/ab/SpecificationMetadata/">
+	<!ENTITY xsd "http://www.w3.org/2001/XMLSchema#">
+]>
+<rdf:RDF xml:base="https://spec.edmcouncil.org/fibo/ontology/FND/AllFND-NorthAmerica/"
+	xmlns:dct="http://purl.org/dc/terms/"
+	xmlns:fibo-fnd-all="https://spec.edmcouncil.org/fibo/ontology/FND/AllFND/"
+	xmlns:fibo-fnd-plc-uspsa="https://spec.edmcouncil.org/fibo/ontology/FND/Places/NorthAmerica/USPostalServiceAddresses/"
+	xmlns:fibo-fnd-plc-uspsai="https://spec.edmcouncil.org/fibo/ontology/FND/Places/NorthAmerica/USPostalServiceAddressesIndividuals/"
+	xmlns:fibo-fnd-utl-av="https://spec.edmcouncil.org/fibo/ontology/FND/Utilities/AnnotationVocabulary/"
+	xmlns:fibo-fndna-all="https://spec.edmcouncil.org/fibo/ontology/FND/AllFND-NorthAmerica/"
+	xmlns:owl="http://www.w3.org/2002/07/owl#"
+	xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+	xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#"
+	xmlns:skos="http://www.w3.org/2004/02/skos/core#"
+	xmlns:sm="http://www.omg.org/techprocess/ab/SpecificationMetadata/"
+	xmlns:xsd="http://www.w3.org/2001/XMLSchema#">
+	
+	<owl:Ontology rdf:about="https://spec.edmcouncil.org/fibo/ontology/FND/AllFND-NorthAmerica/">
+		<rdfs:label>Foundations Domain, North American Extension</rdfs:label>
+		<dct:abstract>The Foundations (FND) domain includes ontologies that define general purpose concepts required to support other FIBO domains. These include concepts and relationships about people, organizations, places, and most importantly, contracts that are essential to domains such as Business Entities (BE), Financial Business and Commerce (FBC), Indices and Indicators (IND), and Securities (SEC). 
```

Comment:
This is not sneaking something in - we discussed moving the annotations to the annotation vocabulary so that they could be more broadly useful, which is what I did.  They could be useful in a number of contexts, in fact.

The other additions are not cities or regions - you are correct, but are important to addressing, regardless of where on a label they occur.

Reply:
Note that for the all file, I followed the same pattern we had used for the others with respect to the abstract, but there is an explanatory note that says what is included.